airbyte-integrations/connectors/source-tiktok-marketing/unit_tests/streams_test.py

```diff
@@ -273,3 +275,12 @@ def test_no_next_page_token(requests_mock):
     requests_mock.get(url, json={"data": {"page_info": {}}})
     test_response = requests.get(url)
     assert stream.next_page_token(test_response) is None
+
+
+@pytest.mark.parametrize(
+    ("original_value", "expected_value"),
+    (("-", None), (26.10, Decimal(26.10)), ("some_str", "some_str")),
```

Comment:
why do we need Decimal package at all?

Reply:
It doesn't make any sense for me either, but this is how the original `transform_function` is written